app/i18n/locales/en.js

```diff
@@ -524,5 +524,7 @@ export default {
 	You_will_be_logged_out_of_this_application: 'You will be logged out of this application.',
 	Clear: 'Clear',
 	This_will_clear_all_your_offline_data: 'This will clear all your offline data.',
-	Mark_unread: 'Mark Unread'
+	Mark_unread: 'Mark Unread',
+	Manual_Authentication_Title: 'Waiting for user to be activated',
```

Comment:
Try to use the same title of `Verify_email` [here](https://github.com/RocketChat/Rocket.Chat.ReactNative/blob/a9e66edc38a6940f73bac00635afbf91d770f20d/app/i18n/locales/en.js#L477), create a new translation and use on both, `Verify email` and `Manual Auth` title.
:)

Reply:
Done